.pre-commit-config.yaml

```diff
@@ -10,6 +10,7 @@ repos:
     rev: 6.1.0
     hooks:
       - id: flake8
+        args: ["--max-line-length=88", "--extend-ignore=E203"]
```

Comment:
why ignore 2023?

Reply:
This is because of the way black formats code in some cases:

> In some cases, as determined by PEP 8, Black will enforce an equal amount of whitespace around slice operators. Due to this, Flake8 will raise `E203 whitespace before ':'` warnings.

https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#flake8